legadilo/templates/base.html

```diff
@@ -66,7 +66,7 @@
             {% endblock page_js %}
         {% endblock javascript %}
     </head>
-    <body class="content-body">
+    <body class="content-body" data-bs-theme="auto">
```

Comment:
Why force this? There is a theme selector in the navbar to choose how to display: automatic theme from system, light or dark. So I don’t think this is necessary.